src/screens/profile/ProfileScreen.tsx

```diff
@@ -56,6 +66,29 @@ export default function ProfileScreen() {
           <H4CardNavTab>{vendorEmail}</H4CardNavTab>
         </LeftAlignContainer>
       </Row>
+      <TouchableOpacity onPress={() => navigation.navigate('ContactUsScreen')}>
+        <Row>
+          <LeftAlignContainer>
+            <Icon2
+              name="phone"
+              size={25}
+              style={Styles.icon}
+              color={Colors.darkGray}
+            />
+          </LeftAlignContainer>
+          <LeftAlignContainer>
+            <ContactUsContainer>
+              <H4CardNavTab>Contact Us</H4CardNavTab>
```

Comment:
Nit: add a space between the phone icon and "Contact us" text.

Reply:
Fixed